spdm.rflx

```diff
@@ -0,0 +1,1487 @@
+            with Size => 8 * 48 * (Slot_Mask_Slot_0_Present
+                                   + Slot_Mask_Slot_1_Present
+                                   + Slot_Mask_Slot_2_Present
+                                   + Slot_Mask_Slot_3_Present
+                                   + Slot_Mask_Slot_4_Present
+                                   + Slot_Mask_Slot_5_Present
+                                   + Slot_Mask_Slot_6_Present
+                                   + Slot_Mask_Slot_7_Present);
+      end message;
+
+   -- 10.8 GET_CERTIFICATE request and CERTIFICATE response messages
+
+   type Slot is
+      (Slot_0              => 16#00#,
+       Slot_1              => 16#01#,
+       Slot_2              => 16#02#,
+       Slot_3              => 16#03#,
+       Slot_4              => 16#04#,
+       Slot_5              => 16#05#,
+       Slot_6              => 16#06#,
+       Slot_7              => 16#07#,
+       Trusted_Environment => 16#FF#)
+   with Size => 8;
+
+   type Offset is mod 2**16;
+
+   type Get_Certificate_Request is
+      message
+         Slot : Slot;
+         Param_2 : Reserved_8;
+         --  This property must be expressed in the session:
+         --     For the first GET_CERTIFICATE request, the Requester shall set this field to 0.
+         --     For non-first requests, Offset is the sum of PortionLength values in all previous
+         --     GET_CERTIFICATE responses.
+         Offset : Offset;
+         Length_1 : Length;
+         Length_2 : Length;
+      end message;
+
+   --  ISSUE: Componolit/RecordFlux#609
+   --  This message differs between SPDM version 1.0 and 1.1.
+   --  Since the minor version is not accessible from
+   --  within this message one message is needed for each version
+   --  that is selected by a refinement.
+   type Certificate_Response_1_0 is
+      message
+         Slot : Slot;
+      end message;
+
+   type Certificate_Response_1_1 is
+      message
+         Slot : Slot;
+         Param_2 : Reserved_8;
+         Portion_Length_1 : Length;
+         Portion_Length_2 : Length;
+         Remainder_Length_1 : Length;
+         Remainder_Length_2 : Length;
+         Cert_Chain : Opaque
+            with Size => 8 * (Portion_Length_2 * 256 + Portion_Length_1);
+      end message;
+
+   -- 10.9 CHALLENGE request and CHALLENGE_AUTH response messages
+
+   type Measurement_Summary_Hash_Type is
+      (No_Measurement   => 16#00#,
+       TCB_Measurement  => 16#01#,
+       All_Measurements => 16#FF#)
+   with Size => 8;
+
+   type Challenge_Request is
+      message
+         Slot : Slot;
+         Hash_Type : Measurement_Summary_Hash_Type;
+         Nonce : Opaque
+            with Size => 8 * 32;
+      end message;
+
+   --  ISSUE: Componolit/RecordFlux#609
+   --  This message differs between SPDM version 1.0 and 1.1.
+   --  Since the minor version is not accessible from
+   --  within this message one message is needed for each version
+   --  that is selected by a refinement.
```

Comment:
As above.

Reply:
Componolit/RecordFlux#609 is fixed. This actually is part of #92 where the duplicate message should be merged again.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/7c1908a0db7eece9cbbb987fa14b0c9d9e4ec91f.